content/docs/concepts/security/index.md

```diff
@@ -224,7 +224,7 @@ Istio provides two types of authentication:
 - **Origin authentication**, also known as **end-user authentication**: verifies the
   original client making the request as an end-user or device.
   Istio enables request-level authentication with JSON Web Token (JWT) validation
-  and a streamlined developer experience for [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/),
+  and a streamlined developer experience for open source OpenID Connect providers [ORY Hydra](https://www.ory.sh), [Keycloak](https://www.keycloak.org), closed source providers [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/),
```

Comment:
I'm not sure if we should explicitly call out OpenID Connect here as we're really allowing **any** JWT. Also I'm not sure if OIDC is the right choice here as we're typically using it on the client side (relying party / RP) as opposed to the resource provider (RP). There we would probably use opaque or transparent bearer tokens such as OAuth 2.0 Access Tokens formatted as random strings (opaque) or as JWTs (transparent).

In fact, I think this is something which is lacking in the documentation of itsio. I'm still getting into the whole topic but am a bit confused about end-user authentication and authorization. This is probably not the right issue to discuss this but I'd be happy to open another one for this.

Reply:
It's a good point that currently Istio only provides JWT authentication. OIDC connect is more on the client side to acquire a token. There is an ongoing work in Istio community to provide OIDC (token acquisition) support at Ingress. For now, it's probably more accurate to say that
"Istio enables request-level authentication with JSON Web Token (JWT) validation for JWTs issued by any OpenID Connect providers, such as [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/), [Google Auth](https://developers.google.com/identity/protocols/OpenIDConnect), [ORY Hydra](https://www.ory.sh), [Keycloak](https://www.keycloak.org). "

I am curious about why you think "JWT authorization is not sufficient for most end-user sessions". Istio authorization allows customers to authorize any JWT claims today. We could chat more on this (my email is liminwang@google.com). Or you can open a GitHub issue for that.